users/serializers.py

```diff
@@ -26,6 +30,23 @@ def get_token(cls: Any, user: Any) -> Any:
         return token
 
 
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ("lookup_id", "username", "email", "password")
```

Comment:
validate email and username are you unique make lookup_id read only, validate password is string eg uppercase lowercase, numbers symbols